src/ape_plugins/_cli.py

```diff
@@ -122,39 +122,73 @@ def _list(cli_ctx, display_all):
             click.echo()
 
 
+def upgrade_option(help: str = ""):
+    """
+    A ``click.option`` for upgrading plugins (``--upgrade``).
+
+    Args:
+        help (str): CLI option help text. Defaults to ``""``.
+    """
+
+    return click.option(
+        "-U",
+        "--upgrade",
+        default=False,
+        is_flag=True,
+        help=help,
+    )
+
+
 @cli.command(short_help="Install an ape plugin")
 @click.argument("plugin")
-@click.option("-v", "--version", help="Specify version (Default is latest)")
+@click.option("--version", help="Specify version (Default is latest)")
 @skip_confirmation_option(help="Don't ask for confirmation to add the plugin")
 @ape_cli_context()
-def add(cli_ctx, plugin, version, skip_confirmation):
+@upgrade_option(help="Upgrade the plugin to the newest available version")
+def add(cli_ctx, plugin, version, skip_confirmation, upgrade):
+    args = [sys.executable, "-m", "pip", "install", "--quiet"]
     if plugin.startswith("ape"):
         cli_ctx.abort(f"Namespace 'ape' in '{plugin}' is not required")
 
     # NOTE: Add namespace prefix (prevents arbitrary installs)
     plugin = f"ape_{clean_plugin_name(plugin)}"
-
     if version:
         plugin = f"{plugin}=={version}"
 
     if plugin in CORE_PLUGINS:
         cli_ctx.abort(f"Cannot add 1st class plugin '{plugin}'")
 
     elif is_plugin_installed(plugin):
-        cli_ctx.abort(f"Plugin '{plugin}' already installed")
+        if upgrade:
+            cli_ctx.logger.info(f"Updating {plugin} plugin.")
```

Comment:
```suggestion
            cli_ctx.logger.info(f"Updating '{plugin}'...")
```

^ just to match line 184.
I love this btw, this is looking great